amd_openvx_extensions/amd_nn/nn_hip/nn_hip_host_decls.h

```diff
@@ -26,6 +26,10 @@ THE SOFTWARE.
 #include "hip/hip_fp16.h"
 #include <VX/vx.h>
 
+int HipExec_tensor_compare_layer(hipStream_t stream, dim3 globalThreads, dim3 localThreads, vx_enum type, uchar* in,
```

Comment:
please remove this line and add it at the end of this file. As a general rule, please add the new functions to the end of this file for better tracking.

Reply:
I thought you were prepending it since gather & tile layers were implemented the latest.
It has now been fixed.